hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java

```diff
@@ -2260,16 +2259,13 @@ public RpcServer(final Server server, final String name,
   }
 
   @Override
-  public void onConfigurationChange(Configuration newConf) {
+  public synchronized void onConfigurationChange(Configuration newConf) {
```

Comment:
If we want to make this change, it should go to master, branch-2 first right?

Reply:
I saw the warning in branch-1 only after HBASE-24174. I can forward-port to branch-2 and master. But as you said it is supressed, dilemma...

Reply:
Right, it is suppressed there. Either we can backport suppressed one or forward port this to branch-2+ and remove suppression there. Sure, let's see QA 👍